packages/nx/src/generators/utils/project-configuration.ts

```diff
@@ -352,8 +352,17 @@ function addProjectToWorkspaceJson(
       if (workspaceConfigPath && mode === 'create') {
         workspaceJson.projects[projectName] = project.root;
       }
+
       // update the project.json file
-      writeJson(tree, configFile, { ...project, root: undefined });
+      const relativeJsonSchemaPath = relative(
```

Comment:
Have we tested that the move generator updates this? It should.

Reply:
Added a test in workspace:move generator